lib/api/apiUtils/apiCallers/callPostObject.js

```diff
@@ -0,0 +1,178 @@
+const { auth, errors } = require('arsenal');
+const busboy = require('@fastify/busboy');
+const writeContinue = require('../../../utilities/writeContinue');
+const fs = require('fs');
+const path = require('path');
+const os = require('os');
+
+// per doc: https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-HTTPPOSTForms.html#HTTPPOSTFormDeclaration
```

Comment:
for jsdoc we could rather write

```suggestion
/** @see https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-HTTPPOSTForms.html#HTTPPOSTFormDeclaration */
```

same below

Reply:
00752ae04313a0bd3319c324106b8cc7dfa0d55f